ui/ducks/send/swap-and-send-utils.ts

```diff
@@ -208,3 +210,50 @@ export async function getSwapAndSendQuotes(request: Request): Promise<Quote[]> {
 
   return newQuotes;
 }
+
+const CHAIN_ID_TO_API_NAME: Record<string, string> = {
+  [CHAIN_IDS.MAINNET]: 'ethereum',
+  [CHAIN_IDS.BSC]: 'bsc',
+  [CHAIN_IDS.POLYGON]: 'polygon',
+  [CHAIN_IDS.AVALANCHE]: 'avalanche',
+  [CHAIN_IDS.ARBITRUM]: 'arbitrum',
+  [CHAIN_IDS.OPTIMISM]: 'optimism',
+  [CHAIN_IDS.ZKSYNC_ERA]: 'zksync',
+  [CHAIN_IDS.LINEA_MAINNET]: 'linea',
+  [CHAIN_IDS.BASE]: 'base',
+};
```

Comment:
I'd recommend reusing swaps util `getNetworkNameByChainId` here to keep things consistent and avoid duplication

Reply:
Good call